cassdegrees/templates/list.html

```diff
@@ -54,10 +57,11 @@
                 </p>
                 <table class="fullwidth tbl-cell-bdr">
                     {# Add title row based on what was in the dictionary #}
-                    <tr>
+                    <thead>
+                        <tr>
                         {% for key in content.0 %}
```

Comment:
Historically this would be indented.

Reply:
Not sure what you mean, it's already indented and I don't think a second indent after the comment is necessary